decisions/models/case.py

```diff
@@ -70,13 +74,15 @@ def __str__(self):
 
 class Content(DataModel):
```

Comment:
Actually, I think it might be better to make `Content` fully translated, so skip translating the fields here for now. In the future, we could add a `language_code` field.